packages/plans-grid-next/src/components/comparison-grid/index.tsx

```diff
@@ -638,18 +667,7 @@ const ComparisonGridFeatureGroupRowCell: React.FunctionComponent< {
 			{ isStorageFeature ? (
 				<>
 					<span className="plan-comparison-grid__plan-title">{ translate( 'Storage' ) }</span>
-					{ canUpgradeStorageForPlan ? (
-						<StorageAddOnDropdown
-							planSlug={ planSlug }
-							storageOptions={ gridPlan.features.storageOptions }
-							onStorageAddOnClick={ onStorageAddOnClick }
-							priceOnSeparateLine
-						/>
-					) : (
-						<StorageButton className="plan-features-2023-grid__storage-button" key={ planSlug }>
-							{ getStorageStringFromFeature( defaultStorageOption?.slug || '' ) }
-						</StorageButton>
-					) }
+					{ storageJSX }
```

Comment:
Thanks for making the nested conditionals more readable 👍

Reply:
doing a lot more of this in https://github.com/Automattic/wp-calypso/pull/91196 and likely others on top